IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_MemberTypeTieBar/Documentation.md

```diff
@@ -0,0 +1 @@
+Properties of catenary tie used in railway. The property set can be used by the predefined type TIEBAR of IfcMember
```

Comment:
is this railway specific description needed suggest to generalise

Reply:
Done. commit : 15b79d25de7519c2d0bf4c1d8a3f413d95e3ee63